### PR TITLE
LPS-100488 | Fix bug in dynamic-data-mapping

### PR DESCRIPTION
**Pull request by kresimir-coko**

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js [hunk 1]
```diff
@@ -4303,6 +4303,7 @@ AUI.add(
 			'liferay-layouts-tree-selectable',
 			'liferay-map-base',
 			'liferay-notice',
+			'liferay-portlet-url',
```
Comment (jbalsas) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js]: Why do we need to add `liferay-portlet-url` back? If we add it here, we should add it in the `config.js` as well, unless we never removed it from there...

Comment (kresimir-coko) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js]: We don't need to add it here, it was for purposes of debugging that I added that and forgotten to remove, I'll force push the code without that change

**Comment by kresimir-coko** [comment]
Hey @jbalsas I've force pushed the change requested

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/77664. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**